imports/plugins/core/job-collection/lib/jobCollection.app-test.js

```diff
@@ -41,6 +41,8 @@ describe("JobCollection", function () {
   let testColl;
 
   before(function () {
+    this.timeout(10000);
```

Comment:
in `accounts.app-test.js`, there was an unscoped before block which set this timeout. accounts.app-test.js does not need a timeout that long, so I moved it here (where some timeouts were observed).